cppcache/integration/test/RegisterKeysTest.cpp

```diff
@@ -39,19 +42,46 @@ namespace {
 using apache::geode::client::binary_semaphore;
 using apache::geode::client::Cache;
 using apache::geode::client::CacheableInt16;
+using apache::geode::client::CacheableInt32;
 using apache::geode::client::CacheableKey;
 using apache::geode::client::CacheableString;
 using apache::geode::client::CacheFactory;
+using apache::geode::client::CacheListener;
 using apache::geode::client::CacheListenerMock;
+using apache::geode::client::EntryEvent;
 using apache::geode::client::IllegalStateException;
 using apache::geode::client::Region;
+using apache::geode::client::RegionEvent;
 using apache::geode::client::RegionShortcut;
 
 using ::testing::_;
 using ::testing::DoAll;
 using ::testing::InvokeWithoutArgs;
 using ::testing::Return;
 
+constexpr auto kNumKeys = 100;
+
+class MyCacheListener : public CacheListener {
+  std::shared_ptr<boost::latch> m_allKeysUpdatedLatch;
```

Comment:
Why are these `std::shared_ptr`? Just make them public and await on them directly?
Also, style guide says `camelCaseFollowedByUnderscore_` for members names.

Reply:
Done